internal/docker/docker.go

```diff
@@ -5,6 +5,7 @@ import (
 	"encoding/base64"
 	"encoding/json"
 	"fmt"
+	"github.com/saucelabs/saucectl/cli/credentials"
```

Comment:
@FriggaHel what do yo think on moving this `import` at the same level as the other saucectl imports?

Reply:
Yup, I did not noticed ! this is corrected !